lib/src/flutterflow_api_client.dart

```diff
@@ -265,3 +279,72 @@ Future _runFix({
     stderr.write('Error running "dart fix": $e\n');
   }
 }
+
+Future firebaseDeploy({
+  required String token,
+  required String projectId,
+  required String destinationPath,
+  String endpoint = kDefaultEndpoint,
+}) async {
+  final endpointUrl = Uri.parse(endpoint);
+  final body = jsonEncode({
+    'project': {
+      'path': 'projects/$projectId',
+    },
+    'token': token,
```

Comment:
Don't pass the token to the body. The API service extracts the token from headers, and it's already included by `_callEndpoint()`. `exportCode` should be updated to not pass token in the request body as well (outside of the scope of this PR).